src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx

```diff
@@ -13,11 +13,17 @@ interface CitySelectProps {
 export const CitySelect = ({ cityValue = 'Москва' }: CitySelectProps): ReactElement => {
   const [showModal, setShowModal] = useState(false);
   const [cityInput, setCityInput] = useState('');
+
   const { selectedEntity: selectedCity, setSelectedEntity: setSelectedCity } = useSetSelectedEntity(
     cityValue,
-    Cookies.City,
+    Cookies.Country,
   );
 
+  // ????
+  // console.log(selectedCity);
+  // const cookGet = getCookie('country');
```

Comment:
yes, this will 1 line will give you the city. But you should only out this 1 string in the middleware, not the entire object. No JSON.parse is needed because there will be just string, not an object